src/ncurses.cr

```diff
@@ -166,5 +169,17 @@ module NCurses
     @@initialized = false
   end
 
+  def ncurses_bits(mask, shift)
```

Comment:
either private, or at least in a more nested namespace, like `Utils` ?
We should ask @jreinert